builder/lxc/config.go

```diff
@@ -29,6 +30,18 @@ type Config struct {
 	ctx interpolate.Context
 }
 
+type ExportConfig struct {
+	Filename string  `mapstructure:"filename"`
```

Comment:
Upper case is fine. I mean that this: 

```Go
type ExportConfig struct {
  Filename string `mapstructure:"filename"`
}
```
will be parsed the same as:
```Go
type ExportConfig struct {
  Filename string
}
```

The mapstructure package will parse and map those names properly because they are the same.